### ISSUE TITLE
Research Previous Documentation on Custom-Scripts and Develop Action Plan for Quarter

### ISSUE THREAD

**Issue by tatmaca** [uchicago-cs/chiventure#1169]
Description: Our last team meeting revealed that we have some legacy code from the last custom-scripts team from two years ago. I'm currently doing some research into this old code and putting together the pros and cons of building off of their work or using their code as a "jumping-off" point for our team to start fresh with all new scripts. From here, we can plan the possible routes our team wants to take moving forward.

This task is complete when I have a comprehensive list of pros and cons regarding the use of the legacy code with references to said code (is the existing code easily readable? does it lend itself to our team building upon it?).

members: Togay

**Comment by tatmaca** [comment]
Description: Our last team meeting revealed that we have some legacy code from the last custom-scripts team from two years ago. I'm currently doing some research into this old code and putting together the pros and cons of building off of their work or using their code as a "jumping-off" point for our team to start fresh with all new scripts. From here, we can plan the possible routes our team wants to take moving forward.

This task is complete when I have a comprehensive list of pros and cons regarding the use of the legacy code with references to said code (is the existing code easily readable? does it lend itself to our team building upon it?).

**Comment by tatmaca** [comment]
Current Iteration of Pros & Cons---will review during Team J meeting
[reLegacyCode.txt](https://github.com/uchicago-cs/chiventure/files/8556768/reLegacyCode.txt)

**Comment by tatmaca** [comment]
Issue completed—after discussing with our team, we decided to build on top of the legacy code. We decided that the prior code, being mostly infrastructural, will allow us to get creative with our own script-writing.

[Additionally, apologies for forgetting to move the card to "in progress" and "done"! It's my first time working with github in this manner and I'll make sure to amend that for next sprint :)]
